torch/_subclasses/meta_utils.py

```diff
@@ -225,6 +225,41 @@ def describe_tensor(self, t: torch.Tensor, recurse: bool = True):
             # sym_sizes_strides_storage_offset (empty_create)
             stride = t.stride()
 
+        # NB: this technically should refer to functorch unwrapped tensor, but
+        # I am (perhaps abusively) using it to store both the functorch and
+        # non-functorch functional tensor
```

Comment:
Is this meant for the `is_functorch_wrapped` check below? If so, could we move it down? (I was confused about what this is pointing to)

Reply:
Oh I see, it's referring to `unwrapped`